modules/microshift-kickstart-prep.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-kickstart-prep.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-kickstart-prep_{context}"]
+= Getting started with the {microshift-short} Kickstart file
+
+Use the Kickstart file provided with the {microshift-short} RPM to provision a virtual machine (VM) by following the instructions for your install type. The ISO you created in the previous steps runs on the VM that you provision with the Kickstart file you create in the following procedures.
+//Do we use the Kickstart to provision a production-version RHEL for Edge also? "Kickstart scripts are plain text files that contain keywords recognized by the installation program, which serve as directions for the installation." from the RHEL docs; so is the installation program in our case image builder?
+
+.Prerequisites
+
+* The machine you are provisioning must meet the system requirements for installing {microshift-short}.
+* The file system must have a logical volume manager (LVM) volume group (VG) with sufficient capacity for the persistent volumes (PVs) of your workload.
+* A pull secret from the https://console.redhat.com/openshift/install/pull-secret[Red Hat Hybrid Cloud Console] must be present as `/etc/crio/openshift-pull-secret` and have root user-only read/write permissions.
+* The firewall must be configured with the required settings.
+//Should we add some guidance on the firewall settings/where to look for that info?
+
+.Procedure
+
+. Install the `microshift-release-info` RPM package containing the sample Kickstart files that are in the `/usr/share/microshift/kickstart` directory by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install -y microshift-release-info
+----
+//Q: is this right? is the kickstart in the release info rpm?
+
+. Install the utilities used during the Kickstart file creation by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install -y openssl gettext
+----
+
+. Set the variables pointing to secrets included in `kickstart.ks`.
+
+* The `PULL_SECRET` file contents are copied to the `/etc/crio/openshift-pull-secret` directory at the post-installation stage to authenticate {ocp} container registry access.
++
+.Example command setting the `PULL_SECRET` variable:
+[source,terminal]
+----
+$ export PULL_SECRET="$(cat ~/.pull-secret.json)" <1>
+----
+<1> Only the encrypted password is included in the Kickstart file.
```

Comment:
This comment is related to `USER_PASSWD`?

Reply:
Hm, I don't remember and it doesn't look right, does it?

Reply:
Added some explainer text. I think this was referring to the encrypted pwd versus the plain text variable value.